nav_msgs/srv/LoadMap.srv

```diff
@@ -0,0 +1,15 @@
+# URL of map resource
+# Can be an absolute path to a file: file:///path/to/maps/floor1.yaml
+# Or, relative to a ROS package: package://my_ros_package/maps/floor2.yaml
+string map_url
```

Comment:
I'd like to posit that **how** the map is loaded or **what** type of file the URL points to is up to the node implementing the service. E.g. it might point to a yaml file containing the metadata, or maybe an image file directly.

See https://github.com/ros-planning/navigation2/pull/1303#discussion_r352957521 for a relevant discussion.

cc @SteveMacenski

Reply:
@jacobperron this is also for ROS1, I'm not as concerned about it.

However you think we can change this to `map_uri`? I typically think of URLs as links to a server rather than a URI which can be a local path.

Reply:
A URL can also be a local path. It describes a way to **locate** a resource.
I'm okay with changing this to a more general URI, but would like to hear what others think. It might make the semantics more ambiguous.

Reply:
I know a URL 'is' that, but in vernacular we usually talk about servers. A URL is a subclass of URI. 100% agree its semantics. I just think it makes it more clear actually because of the vernacular uses and URI is what is used in SDF for getting models/packages/files in the same way this is attempting to so its consistent with other ROS-y tools.

I don't care that much one way or another since I know what it means, but I could see someone just looking at a message definition in `ros2 msg show ...` and thinking it _had_ to be a web link